internal/tss/channels/error_non_jitter_channel.go

```diff
@@ -0,0 +1,92 @@
+package channels
+
+import (
+	"context"
+	"slices"
+	"time"
+
+	"github.com/alitto/pond"
+	"github.com/stellar/go/support/log"
+	"github.com/stellar/wallet-backend/internal/tss"
+	"github.com/stellar/wallet-backend/internal/tss/router"
+	"github.com/stellar/wallet-backend/internal/tss/services"
+	tss_store "github.com/stellar/wallet-backend/internal/tss/store"
+)
+
+type ErrorNonJitterChannelConfigs struct {
+	TxManager         services.TransactionManager
+	Router            router.Router
+	MaxBufferSize     int
+	MaxWorkers        int
+	MaxRetries        int
+	WaitBtwnRetriesMS int
+}
+
+type errorNonJitterPool struct {
+	Pool              *pond.WorkerPool
+	TxManager         services.TransactionManager
+	Store             tss_store.Store
+	Router            router.Router
+	MaxRetries        int
+	WaitBtwnRetriesMS int
+}
+
+var ErrorNonJitterChannelName = "ErrorNonJitterChannel"
+
+var _ tss.Channel = (*errorNonJitterPool)(nil)
+
+func NewErrorNonJitterChannel(cfg ErrorNonJitterChannelConfigs) *errorNonJitterPool {
+	pool := pond.New(cfg.MaxBufferSize, cfg.MaxWorkers, pond.Strategy(pond.Balanced()))
+	return &errorNonJitterPool{
+		Pool:              pool,
+		TxManager:         cfg.TxManager,
+		Router:            cfg.Router,
+		MaxRetries:        cfg.MaxRetries,
+		WaitBtwnRetriesMS: cfg.WaitBtwnRetriesMS,
+	}
+}
+
+func (p *errorNonJitterPool) Send(payload tss.Payload) {
+	p.Pool.Submit(func() {
+		p.Receive(payload)
+	})
+}
+
+func (p *errorNonJitterPool) Receive(payload tss.Payload) {
+	ctx := context.Background()
+	var i int
+	for i = 0; i < p.MaxRetries; i++ {
+		time.Sleep(time.Duration(p.WaitBtwnRetriesMS) * time.Microsecond)
```

Comment:
Please see comments from the jitter channel, most would apply here too

Reply:
Yeap made changes in both